src/components/Tabs/Tabs.module.scss

```diff
@@ -10,9 +10,27 @@ $theme-grey: #d4d4d5;
     cursor: pointer;
 }
 
+.featureFlagTabButton {
+    background: white;
+    border: none;
+    padding: 12px 8px 8px;
+    font-size: 0.8em;
+    border: none;
+    cursor: pointer;
+    color: #a1a5b7;
```

Comment:
Would it be better to create variable name for color?

Reply:
I didn't use a variable name since it's being used only once. But if it's better, then I'll surely use it.